tests/main.cpp

```diff
@@ -1,8 +1,73 @@
+#include <chrono>
+#include <iostream>
+#include <string>
+#include <thread>
+
+#include "solana.hpp"
 #define DOCTEST_CONFIG_IMPLEMENT_WITH_MAIN
 #include <doctest/doctest.h>
 
 #include "MangoAccount.hpp"
 
+const std::string DEVNET = "https://api.devnet.solana.com";
```

Comment:
move this definition to solana.hpp

https://github.com/mschneider/solcpp/pull/42/files/6934148919f55fb4e612bb519aa7dc85a0828c43#diff-daa5fea3872a516a483020907c6131367cbb3aacdc61e601205ac2b8a22448a3R22